front/cypress/e2e/admin/pages_and_menu/update_homepage_text_content_and_sections.cy.ts

```diff
@@ -22,6 +22,7 @@ describe('Admin: update text content and sections', () => {
     cy.intercept('PATCH', '**/home_page').as('saveHomePage');
     cy.intercept('GET', '**/home_page').as('getHomePage');
     cy.intercept('GET', '**/pages-menu').as('getPages');
+    cy.intercept('GET', '**/nav_bar_items').as('getNavbarItems');
```

Comment:
The page is not rendered if this request is not fulfilled so hoping this will resolve the flakiness 🤞